### ISSUE TITLE
trailing comments on fields supported?

### ISSUE THREAD

**Issue by quintilation** [yoheimuta/protolint#292]
Hi,

We have a lot of comments like this:

message xyz {
       string a = 1;                 // first arg

      // next arg
       bool  b = 2;
}

But protolint is complaining the field a does not have a comment?
Have I missed a trick or are only preceeding/prefix comments supported?

Thanks,
-Steve

**Comment by yoheimuta** [comment]
@quintilation Thank you for telling me.

No. protolint's `HAVE_COMMENT` rules don't support trailing comments.
And your question makes sense. 
I realize the support would be worth it, so I'm getting to it soon.

**Comment by yoheimuta** [comment]
@quintilation I've cut a release of [v0.42.0](https://github.com/yoheimuta/protolint/releases/tag/v0.42.0).
Can you give it a try?

**Comment by quintilation** [comment]
Thanks very much, this almost does what we would like...
We have one case which fails in our proto files, rpc statments.
These again have trailing comments and are not found by protolint

```
	rpc GetAll(GetRequest) returns(GetReply) {			// get the global address table
		option(requestreply.Nats).Subject = "get.addrs";
	}

produces
            [addrs/addrs.proto:44:2] RPC "GetAll" should have a comment

```

**Comment by yoheimuta** [comment]
Thank you for letting me know. I'm working on it.

**Comment by yoheimuta** [comment]
@quintilation I fixed it. Could you please install [v0.42.1](https://github.com/yoheimuta/protolint/releases/tag/v0.42.1)?

**Comment by yoheimuta** [comment]
I fixed it and some time passed, so I'm closing this issue.
